### DISCUSSION TITLE
Releases

### DISCUSSION THREAD

**Discussion by devcodeman** [nasa/cFS#809] [Q&A] [answered]
Hello, I am working an IRAD for my company and interested in using cFS. I have noticed that the releases have been pretty stale compared to F'. Is this just a simple oversight, or has NASA moved on from cFS towards F'?

**Comment by ivanperez-keera** [comment]
@cdeeran cFS is actively developed. Both large and new NASA missions keep using cFS.

**Comment by RichLandau** [comment, marked as answer]
Welcome to the community @cdeeran. Lots of cFS development is ongoing throughout the Agency and community. There have been internal delays in creating new "official" releases which is being worked, but release candidates exist with the latest features. There are also features being developed internally that cannot be released but partnerships are possible with the cFS team if your company is interested in pursuing that route.

**Comment by irowebbn** [comment]
One thing to note is that although the last release marked as "official" in the GitHub is old, there have been a number of pre-release versions, or "release candidates", published since then. The most recent is [Equuleus-rc1](https://github.com/nasa/cFS/releases/tag/equuleus-rc1). My understanding is that these cannot be marked as official releases yet due to the constraints of the NASA open source review process, but I can't speak on behalf of the cFS team about that.

F' does seem to publish releases far more frequently, but cFS is very much in use at NASA.

**Comment by devcodeman** [comment]
Thank you for all of your replies. This is very helpful.
